README.md

```diff
@@ -10,7 +10,14 @@
 
 ### Run as a Spring Boot application
 
-`mvn clean install spring-boot:run`
+Using local profile:
+`mvn clean install spring-boot:run -Dspring.profiles.active=local`
+
+### Run with MongoDB
+This application uses MongoDB as it's datastore, to run MongoDB:
+
+1. Deploy an instance of MongoDB as a docker container `docker run -p 27017:27017 --name mongodb -d mongo:4.2` 
```

Comment:
I will try to figure out what version of MongoDB we should be coding against.

Reply:
A `3.4.2` is used in local dev environment so we might use that one. Trying to figure out what is deployed in other envs and how much it matters.

Reply:
I suggest we use latest from 3.6 branch as this is used in dev environments.